requirements.txt

```diff
@@ -17,3 +17,4 @@ typer>=0.9.0
 mlflow>=2.10.0
 psutil>=5.0.0
 tabulate>=0.9.0
+grad-cam>=1.5.4
```

Comment:
I wouldn't have always required. Do we want to put this requirement directly in the class? CC: also @kozlov721

Reply:
Yeah, moving it to the class makes sense

Reply:
IMHO we don't really need to care about optimizing the dependencies of `luxonis-train` because it will be huge anyway